prow_config.yaml

```diff
@@ -4,3 +4,20 @@ workflows:
   - app_dir: kubeflow/testing/workflows
     component: workflows
     name: unittests
+
+  - app_dir: kubeflow/kubeflow/testing/workflows
+    component: click_deploy_test
+    name: skipmeplease
+    job_types:
+    - periodic
```

Comment:
This should be presubmit.
If you set it to periodic then you aren't testing whether its being skipped because no matching files were modified.

Reply:
Fixed.